app/src/examples/ScreenAnimationExample.tsx

```diff
@@ -0,0 +1,96 @@
+// @ts-nocheck - It will be removed after release of rn-screens, because currently 'react-native-screens/gesture-handler' import doesn't exist.
+import React from 'react';
+import { View, StyleSheet, Button } from 'react-native';
+import {
+  createNativeStackNavigator,
+  NativeStackScreenProps,
+} from 'react-native-screens/native-stack';
+import { GestureDetectorProvider } from 'react-native-screens/gesture-handler';
+import { ParamListBase } from '@react-navigation/native';
+import {
+  ScreenTransition,
+  AnimatedScreenTransition,
+} from 'react-native-reanimated';
+
+const MainScreen = ({ navigation }: NativeStackScreenProps<ParamListBase>) => (
```

Comment:
Throughout our codebase we're mostly using function declaration syntax for React components (and screens).
```suggestion
function MainScreen({ navigation }: NativeStackScreenProps<ParamListBase>) {
```